tests/fixtures/odl.py

```diff
@@ -217,167 +205,3 @@ def opds2_with_odl_api_fixture(
     opds2_with_odl_files_fixture: OPDS2WithODLFilesFixture,
 ) -> OPDS2WithODLApiFixture:
     return OPDS2WithODLApiFixture(db, http_client, opds2_with_odl_files_fixture)
-
-
-class LicenseHelper:
```

Comment:
Instead of these custom helper classes, I re-wrote the code using them to use the OPDS model classes instead while I was updating the tests for this.